templates/outing/index.html.twig

```diff
@@ -51,9 +51,11 @@
                     <a href="{{ path('outing_edit', {'id': outing.id}) }}">edit</a>
                     {% endif %}
                     {% if (date(outing.registrationDeadLine) > date()) %}
-                        {% include "outing/_register_form.html.twig" %}
+                        {% if app.user not in outing.participants.toArray and outing.state.id == 2 %}
+                            {% include "outing/_register_form.html.twig" %}
+                        {% endif %}
```

Comment:
`outing.state.id == 2`

j'aurais mis

`outing.state.id is same as(2)`

Reply:
idem
Et tu n'as pas besoin de faire un toArray car c'est déjà un tableau. En tout cas ça n'a pas bloqué quand Mikael a fait le unregister.